e2e/jest.test-setup.ts

```diff
@@ -128,12 +128,24 @@ beforeEach(async () => {
   });
 
   page.on('requestfinished', async (request) => {
-    if (canLogResponse(request)) {
-      if (isApiFailure(request)) {
-        await logError(request);
-      } else {
-        await logResponse(request);
+    let method;
+    let url;
+    let status;
+    try {
+      if (canLogResponse(request)) {
+        method = request.method();
+        const resp = request.response();
+        url = resp.url();
+        status = resp.status();
+        if (isApiFailure(request)) {
+          await logError(request);
+        } else {
+          await logResponse(request);
+        }
       }
+    } catch (err) {
+      // Try find out what request was
+      console.error(`${status} ${method} ${url} : ${err}`);
```

Comment:
Exception caused by parsing very long console message. Simply print out the original long string to find out what the console message was.
```
Exception occurred when getting console messages.
      Error: Protocol error (Runtime.callFunctionOn): Object reference chain is too long

      153 |       // tslint:disable-next-line:no-empty
      154 |     } catch (err) {
    > 155 |       console.error(`Exception occurred when getting console messages.\n${err}`);
          |               ^
      156 |     }
      157 |   });
```